org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/SignatureUtils.java

```diff
@@ -0,0 +1,229 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.sun.jdi.ClassLoaderReference;
+import com.sun.jdi.ReferenceType;
+import com.sun.jdi.Type;
+import com.sun.jdi.VirtualMachine;
+
+public abstract class SignatureUtils {
+    static final char ARRAY = '[';
+    static final char BYTE = 'B';
```

Comment:
org.eclipse.jdt.core.Signature has well defined constant values for these. Can we reuse that part?